### PR TITLE
Feat: moving windows whether they are floating or tiled, using the same keybind.

### PR DESCRIPTION
**Pull request by Lol-Lap**
## Description
Script for moving windows, whether they are floating or tiled, using the same keybinds.

## Type of change

Please put an `x` in the boxes that apply:

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [x] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [x] **Documentation update** (non-breaking change; modified files are limited to the documentations)
- [ ] **Technical debt** (a code change that does not fix a bug or add a feature but makes something clearer for devs)
- [ ] **Other** (provide details below)

## Checklist

Please put an `x` in the boxes that apply:

- [x] I have read the [CONTRIBUTING](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] My commit message follows the [commit guidelines](https://github.com/prasanthrangan/hyprdots/blob/main/CONTRIBUTING.md#git-commit-messages).
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added a changelog entry.
- [x] I have added necessary comments/documentation to my code.
- [ ] I have added tests to cover my changes.
- [x] I have tested my code locally and it works as expected.
- [x] All new and existing tests passed.

## Screenshots

(if appropriate)

### CODE REVIEW

**Review by kRHYME7 — CHANGES_REQUESTED** [review]
We can just use hyprlands capability to call shell funtions

We can remove this script

File: Configs/.config/hypr/keybindings.conf [hunk 1]
```diff
@@ -129,10 +133,10 @@ bind = $mainMod+Ctrl+Alt, Right, movetoworkspace, r+1
 bind = $mainMod+Ctrl+Alt, Left, movetoworkspace, r-1
 
 # Move focused window around the current workspace
-bind = $mainMod+Shift+Ctrl, Left, movewindow, l
```
Comment (kRHYME7) [inline Configs/.config/hypr/keybindings.conf:132]: Simply we can just use this. try it

```
# Move active window around current workspace with mainMod + SHIFT + CTRL [←→↑↓]
$moveactivewindow=grep -q "true" <<< $(hyprctl activewindow -j | jq -r .floating) && hyprctl dispatch moveactive
binded = $mainMod SHIFT $CONTROL, left,Move activewindow to the right,exec, $moveactivewindow -30 0 || hyprctl dispatch movewindow l
binded = $mainMod SHIFT $CONTROL, right,Move activewindow to the right,exec, $moveactivewindow 30 0 || hyprctl dispatch movewindow r
binded = $mainMod SHIFT $CONTROL, up,Move activewindow to the right,exec, $moveactivewindow  0 -30 || hyprctl dispatch movewindow u
binded = $mainMod SHIFT $CONTROL, down,Move activewindow to the right,exec, $moveactivewindow 0 30 || hyprctl dispatch movewindow d
```

File: README.md
Comment (kRHYME7) [inline README.md:1]: Cool

File: Configs/.local/share/bin/move-window.sh
Comment (kRHYME7) [inline Configs/.local/share/bin/move-window.sh:1]: We might don't need this anymore

**Review by kRHYME7 — CHANGES_REQUESTED** [review]
Thank you!

**Comment by kRHYME7** [comment]
https://github.com/user-attachments/assets/2926c655-ae1d-416f-8c0b-74d500d84905

File: Configs/.config/hypr/keybindings.conf (hunk 1 above)
Comment (Lol-Lap) [reply Configs/.config/hypr/keybindings.conf:132]: > Simply we can just use this. try it
> 
> ```
> # Move active window around current workspace with mainMod + SHIFT + CTRL [←→↑↓]
> $moveactivewindow=grep -q "true" <<< $(hyprctl activewindow -j | jq -r .floating) && hyprctl dispatch moveactive
> binded = $mainMod SHIFT $CONTROL, left,Move activewindow to the right,exec, $moveactivewindow -30 0 || hyprctl dispatch movewindow l
> binded = $mainMod SHIFT $CONTROL, right,Move activewindow to the right,exec, $moveactivewindow 30 0 || hyprctl dispatch movewindow r
> binded = $mainMod SHIFT $CONTROL, up,Move activewindow to the right,exec, $moveactivewindow  0 -30 || hyprctl dispatch movewindow u
> binded = $mainMod SHIFT $CONTROL, down,Move activewindow to the right,exec, $moveactivewindow 0 30 || hyprctl dispatch movewindow d
> ```

That's so cool! Thanks.

**Review by kRHYME7 — APPROVED** [review]
Should be great. 💫

**Pull request merged**